msal/src/test/java/com/microsoft/identity/nativeauth/NativeAuthPublicClientApplicationJavaTest.java

```diff
@@ -258,7 +270,11 @@ public void onError(@NonNull BaseException exception) {
         };
         application.signInUsingPassword(username, password, null, callback);
         // 1a. Server returns invalid password error
-        assertTrue(signInResult.get(30, TimeUnit.SECONDS) instanceof SignInResult.InvalidCredentials);
+        SignInUsingPasswordResult result = signInResult.get(30, TimeUnit.SECONDS);
+        assertTrue(result instanceof SignInUsingPasswordError);
```

Comment:
We should avoid using `assertTrue` where we are no comparing a boolean object. If `assertTrue` fails in this case then we won't know from the test run about the actual class  of `result`. Below shows an example using `assertThat`

https://stackoverflow.com/questions/12404650/assert-an-object-is-a-specific-type